cfme/scripting/sprout.py

```diff
@@ -41,14 +41,21 @@ def main():
 @click.option('--populate-yaml', is_flag=True, default=False,
               help="Populate the yaml with the appliance")
 @click.option('--provider', default=None, help="Which provider to use")
+@click.option('--provider-type', 'provider_type', default=None,
+              help='A provider type to select from')
+@click.option('--template-type', 'template_type', default=None, help='A template type')
+@click.option('--preconfigured', default=True, help='Whether the appliance is configured')
```

Comment:
Question: will this option have False if it is used ?

Reply:
@izapolsk Good call, I'll actually add a --notconfigured false boolean, so its an explicit flag.